yarn.lock

```diff
@@ -5011,6 +5028,14 @@ react@=0.14.7:
     envify "^3.0.0"
     fbjs "^0.6.1"
 
+react@15.3.1:
```

Comment:
yeah, that's interesting. I just looked an it seems like it's installing both 14.7 and 15.3. It installs 15.3 so that flow-coverage-reporter can use it. I believe this is okay.